lib/ecmascript.ts

```diff
@@ -6139,24 +6201,15 @@ function GetOption<
   return fallback;
 }
 
-function GetNumberOption<P extends StringlyTypedKeys<T>, T extends Partial<Record<P, unknown>>>(
-  options: T,
-  property: P,
-  minimum: number,
-  maximum: number,
-  fallback: number
-): number {
-  let valueRaw = options[property];
-  if (valueRaw === undefined) return fallback;
-  const value = ToNumber(valueRaw);
-  if (NumberIsNaN(value) || value < minimum || value > maximum) {
-    throw new RangeError(`${String(property)} must be between ${minimum} and ${maximum}, not ${value}`);
-  }
-  return MathFloor(value);
+export function IsBuiltinCalendar(id: string): id is BuiltinCalendarId {
+  return BUILTIN_CALENDAR_IDS.includes(ASCIILowercase(id));
 }
 
-export function IsBuiltinCalendar(id: string): id is BuiltinCalendarId {
-  return ArrayIncludes.call(BUILTIN_CALENDAR_IDS, id);
+export function ASCIILowercase<T extends string>(str: T): T {
```

Comment:
Why not just .lowercase() ? If non-ASCII unicode is supposed to be left alone, can we at least document why?

Reply:
Decided to do this comment upstream. https://github.com/tc39/proposal-temporal/pull/2559